lib/datasource/deb/index.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { copyFile, mkdirp, stat } from 'fs-extra';
```

Comment:
file structure changed, please move datasource to right folder.

please don't force-push